client/main.lua

```diff
@@ -219,15 +220,22 @@ function createZones()
                     inNoDispatchZone = false
                 end
             })
+            table.insert(nodispatchzones, tmp)
     	end
     end
 end
 
 local function removeZones()
```

Comment:
Hi, include a nil check to ensure that the zones are not nil when being removed for good practice

Reply:
Thanks so much!